us-geological-survey.csl

```diff
@@ -121,8 +150,8 @@
               <et-al term="and others"/>
               <label form="short" prefix=" "/>
             </names>
-            <text variable="container-title" prefix=" " suffix=","/>
-            <text variable="collection-title" prefix=" " suffix=","/>
+            <text variable="event" prefix=" " suffix=" --"/>
```

Comment:
Hey Sebastian,

I don't recognize that slice of code or suffix...I focused mainly on the
sections related to report and webpage documents. I'm no longer in front of
my computer and won't be so for a week - I'll have a look when I get back.

On Friday, March 20, 2015, Sebastian Karcher <notifications@github.com>
wrote:

> In us-geological-survey.csl
> <https://github.com/citation-style-language/styles/pull/1463#discussion_r26870609>
> :
>
> > @@ -121,8 +150,8 @@
> >                <et-al term="and others"/>
> >                <label form="short" prefix=" "/>
> >              </names>
> > -            <text variable="container-title" prefix=" " suffix=","/>
> > -            <text variable="collection-title" prefix=" " suffix=","/>
> > +            <text variable="event" prefix=" " suffix=" --"/>
>
> the style really wants two ugly hyphens there? not a pretty little en dash
> or so?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/citation-style-language/styles/pull/1463/files#r26870609>
> .
>


-- 
Sent from Gmail Mobile

Jason C. Bellino
jbellino@usgs.gov
US Geological Survey
4446 Pet Ln
Lutz, FL 33559